sources/presentation/Stride.Core.Presentation/Themes/ThemeType.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Stride contributors (https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Stride.Core.Presentation.Themes
+{
+    public enum ThemeType
+    {
+        // Dark themes
+        [Display("Expression Dark (Default)")]
+        ExpressionDark,
+        [Display("Dark Steel")]
+        DarkSteel,
+
+        // Light themes
+        [Display("Light Steel Blue")]
```

Comment:
If the light theme is not perfect/complete, maybe we could change display name to `Light Steel Blue (Experimental)`?

Reply:
Yep, I'm ok with that. Done and rebased on top of the latest (this was the only change, apologies if you didn't want me to rebase again).

Reply:
Thanks for rebasing, it's helpful!